com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/RenderPass/AOV/AOVRequestData.cs

```diff
@@ -59,6 +59,7 @@ public struct AOVRequestData
         private readonly AOVRequestBufferAllocator m_BufferAllocator;
         private readonly AOVRequestCustomPassBufferAllocator m_CustomPassBufferAllocator;
         private List<GameObject> m_LightFilter;
+        private BufferedRTHandleSystem m_HistoryRTSystem; // Each AOV render request gets a separate set of history buffers (to avoid mixing history data between different AOVs)
```

Comment:
I think you need to manually call dispose on the system like we do in HDCamera otherwise it can leak render textures.

Reply:
Yes you are right, I'm looking now how to properly do this (I will probably move the AOV m_HistoryRTSystem to a static dictionary).